VL.Data/Services/BookService.cs

```diff
@@ -0,0 +1,63 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using VL.Shared.Data;
+using VL.Shared.Interfaces;
+using VL.Shared.Model;
+
+namespace VL.Shared.Services
+{
+    public class BookService : IBookService
+    {
+        private readonly ApplicationDbContext _applicationDbContext;
+        private readonly IMapper _mapper;
+
+        public BookService(ApplicationDbContext applicationDbContext, IMapper mapper)
+        {
+            _applicationDbContext = applicationDbContext;
+            _mapper = mapper;
+        }
+
+        public async Task<IEnumerable<BookDto>> GetBooksAsync()
+        {
+            var bookList = await _applicationDbContext.Book
```

Comment:
```
return await _applicationDbContext.Book
    .ProjectTo<BookDto>(_mapper.ConfigurationProvider)
    .ToListAsync();
```

Reply:
Returns a list of books, mapped to BookDto

If you want to keep a variable for it, it would just be `var bookDtos = ...`